### PR TITLE
last couple of tweaks

### PR DESCRIPTION
**Pull request by alice**

### CODE REVIEW

File: docs/blog.md [hunk 1]
```diff
@@ -25,7 +25,9 @@ Then, of course, there are users who use the keyboard heavily or exclusively. Th
  
 In any case, the `tab` key allows users to navigate sequentially through interactive elements on the page (such as form fields) with just a keyboard. The _only way_ for keyboard users to know which element is active (and thus how they might potentially interact with it via the keyboard) is via a visual indictor of what thing is focused - i.e., by default, the __focus ring__).
 
-Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, since designs vary, what the particular focus ring should look like could vary too. On a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, it might be necessary to consider this part of branding and perhaps provide something like:
+Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, this runs into issues before we even consider user context.
```
Comment (bkardell) [inline docs/blog.md]: I don't like how this hangs - can we maybe just say "However, this quickly runs into issues".  I think "before we even consider" makes it a little more confusing than it needs to be.

Comment (alice) [inline docs/blog.md]: Yeah I can see that - I'm trying to distinguish between the issues we're trying to solve and the issue that's already been solved (styling focus). Any ideas which let us keep that?

Comment (bkardell) [inline docs/blog.md]: Well, it seems like we are kind of making the case that styling focus actually isn't incompletely solved - that it's problematic.  Maybe "However this quickly runs into issues, even in really basic, long-standing cases"

Comment (alice) [inline docs/blog.md]: SGTM, will fix.

**Pull request merged**